scopes/defender/jest/jest.worker.ts

```diff
@@ -35,6 +39,10 @@ export class JestWorker {
             `${__dirname}/watch.js`,
             {
               specFiles: testFiles,
+              shouldRunTestSuite: async (specFile: string) => {
+                if (!this.shouldRunTestSuite) return true;
```

Comment:
this is not good, in case of false you will get true as well. you probably want to check if it's undefined so something